src/helpers/replaceWithCurrentDomain.js

```diff
@@ -0,0 +1,8 @@
+// @flow
```

Comment:
Can we group both helpers in one file? They seem quite related :slightly_smiling_face:

Reply:
¡Claro! Should be cool now :wink: